hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/systemcontracts/ExchangeRateSystemContract.java

```diff
@@ -63,7 +63,6 @@ public FullResult computeFully(@NonNull Bytes input, @NonNull MessageFrame messa
         requireNonNull(messageFrame);
         try {
             validateTrue(input.size() >= 4, INVALID_TRANSACTION_BODY);
-            validateTrue(messageFrame.getValue().getAsBigInteger().equals(BigInteger.ZERO), INVALID_FEE_SUBMITTED);
```

Comment:
What about L82-83?  Unneeded now?

Reply:
And see also `ExchangeRateSystemContractTest.valueShouldNotBeSentToThePrecompile` - doesn't seem still applicable?

Reply:
good point.  removed both.